README.md

```diff
@@ -16,28 +16,33 @@ This is the list with latest details on the OSs Equinix Metal repackages and pub
-| CentOS                  | 7     | Jun, 2024       | Mar, 2021  | [x86_64](centos/x86_64/7.md)                                      | |
-| AlmaLinux OS            | 8     | May, 2024       | Feb, 2024  | [x86_64](alma/x86_64/8.md) [arm64](alma/aarch64/8.md)             | |
 | AlmaLinux OS            | 9     | May, 2027       | Feb, 2027  | [x86_64](alma/x86_64/9.md) [arm64](alma/aarch64/9.md)             | |
-| Rocky Linux             | 8     | May, 2024       | Feb, 2024  | [x86_64](rocky/x86_64/8.md) [arm64](rocky/aarch64/8.md)           | |
 | Rocky Linux             | 9     | May, 2025       | Feb, 2025  | [x86_64](rocky/x86_64/9.md) [arm64](rocky/aarch64/9.md)           | |
 | Alpine                  | 3     | Rolling Release | N/A        | [x86_64](alpine/x86_64/3.md) [arm64](alpine/aarch64/3.md)         | |
 | Nixos                   | 22.05, 22.11 | June, 2023 | Alongside upstream      | TBD                                                | |
-| VMware ESXi             | 6.7   | Oct, 2022       | Aug, 2022  | N/A          | Not available on latest HW. No new images published. |
 | VMware ESXi             | 7.0   | Apr, 2025       | Jan, 2025  | TBD                                                               | |
 | Windows Server          | 2022  | Oct, 2026       |            | [x86_64](windows%202022/windows_2022.md)                                  | |
-| Windows Server          | 2019  | Jan, 2024       | Sept, 2023 | [x86_64](windows%202019/windows_2019.md)                                  | |
 | Talos                   | v1    | Rolling Release | N/A        | [x86_64](talos/talos_v1.md) [arm64](talos/talos_v1.md) | Available | |
 | Flatcar Linux           |stable | Rolling release | N/A        | Rolling release                       | Available                 | |
 | Flatcar Linux           |beta   | Rolling release | N/A        | Rolling release                       | Available                 | |
 | FreeBSD                 | 13.2  | Jan, 2026       | Jan, 2026  | [x86_64](freebsd/x86_64/13_2.md) [arm64](freebsd/arm64/13_2.md)   | |
 | FreeBSD                 | 14.0  | Jan, 2029       | Jan, 2029  | [x86_64](freebsd/x86_64/14_0.md) [arm64](freebsd/arm64/14_0.md)   | |
 | VyOS                    | 1.3   | ???             | ???        | TBD                                   | Coming soon               | |
+
+## EOL Operating Systems
+
+The following OSes have been officialy EOL'ed at Equinix Metal and are no longer available. The Changelogs for these OSes have been deleted from this repository (but can still be retrieved via the commit history).
+
+| OS                      | Version |
+| ---                     | ---   |
+| AlmaLinux OS            | 8     |
+| CentOS                  | 7,8   |
+| Debian                  | 10    |
+| RedHat Enterprise Linux | 7,8   |
+| Rocky Linux             | 8     |
+| Ubuntu                  | 16.04,18.04 |
+| VMware ESXi             | 6.7   |
+| Windows Server          | 2019  |
```

Comment:
Windows 2019 is still under Extended Support -- https://learn.microsoft.com/en-us/lifecycle/products/windows-server-2019
https://learn.microsoft.com/en-us/lifecycle/policies/fixed#extended-support

Reply:
Fixed, thanks!